### ISSUE TITLE
Incorrect `write` implementation on Windows

### ISSUE THREAD

**Issue by kennethnym** [mogol/flutter_secure_storage#275]
I just realized after reading the docs that `write` accepts `null` value. If it receives `null` it deletes the key instead. Originally I assumed value will never be null, and so there isn't any null check done on the value. I will make another PR after #274 is merged.

**Comment by kennethnym** [comment]
Or I can lump it to #274 instead.

**Comment by juliansteenbakker** [comment]
If you want you can add it to #274
